### ISSUE TITLE
Usage with Backbone.Marionette

### ISSUE THREAD

**Issue by cosmicnag** [pwalczyszyn/backstack#20]
Hi. This looks awesome. Whats the best way to use it if I am using Marionette on top of Backbone ?

**Comment by pwalczyszyn** [comment]
Sorry, I have no expiriance with Marionette ;(

**Comment by cosmicnag** [comment]
No problem... Will update you about Marionette as I intend to try it out soon. Should somehow work as Marionettes 'classes' basically extend Backbone.View .

**Comment by lardissone** [comment]
@cosmicnag did you get it work with Marionette?

**Comment by cosmicnag** [comment]
@lardissone nope...i found out that i can use marionettes regions along with the backbone router to achieve what i need without other libraries.cheers.

**Comment by jarrodpayne** [comment]
@cosmicnag Could you elaborate on your implementation? I am using Marionette Regions as well, but don't see a clear path to transitioning between region replacement. I'm thinking I need to duplicate the region, transition the original out, while transitioning the new region in before the new region actually replaces the old.

**Comment by cosmicnag** [comment]
@jpdesigndev You mean animated transitions ? I havent (yet) required to animate transitions... but this way (see the accepted answer) seems like it should do the trick.
http://stackoverflow.com/questions/11673445/backbone-marionette-add-a-visual-effect-when-switching-view

**Comment by jarrodpayne** [comment]
@cosmicnag Thanks! Not sure how I missed this response.
